src/main/python/pgshovel/replication/streams/__init__.py

```diff
@@ -0,0 +1,6 @@
+from pgshovel.replication.validation import validate_state
+
+
+#: The expected types of event for a stream of transactions when there is no
+#: existing ``TransactionState``.
+TRANSACTION_START_EVENT_TYPES = validate_state.validators['stream_state'].validators['transaction_state'].receivers[None].keys()  # noqa
```

Comment:
Not stupid, but I'd probably keep this near the definition of the validators instead of in `__init__.py`.